### PR TITLE
chore: remove stale bot

### PR DESCRIPTION
**Pull request by jaworek**
### Summary

Inspired by blog post linked below, I would like to propose removing stale bot from the project.
I'll quickly summarise points mentioned there:
- opened issue should not make maintainer feel obligated to respond, implement, or fix any feature/bug
- issues are a place for discussion and a way for community to find solutions to problems that they encounter
- it serves as a place to record additional information, workaround, opportunity for other developers to create a PR in the future

https://drewdevault.com/2021/10/26/stalebot.html

I'm aware that we also have a "Discussions" enabled for this reason, but it seems as it is not uses as often as "Issues" tab.

### Test plan

Verify that merge and build pipeline is working correctly.

### CODE REVIEW

**Review by BartoszKlonowski — APPROVED** [review]
In my opinion it's always good, polite, professional to respond to an issue, in any kind of a response, whether it's adding a label, or writing a comment.
I'm trying to send responses in Slider when there's a new issue, or at least apply a label.
I believe that it let's the reporting user know he's not completely ignored.
That's why I only partially agree with the linked article.

**I do, however, believe that an issue is also a place to discuss things, share solutions, not only to complain about bugs and blame the owner/owners.**

So I think that it's best to let the author close it when he/she gets the answer or decide that issue can/should be closed.
I wouldn't like to have my issue closed only due to a given time of inactivity, especially when it's an automated bot, not a human who closes it just like that.

**Comment by satya164** [comment]
> - opened issue should not make maintainer feel obligated to respond, implement, or fix any feature/bug
> - issues are a place for discussion and a way for community to find solutions to problems that they encounter
> - it serves as a place to record additional information, workaround, opportunity for other developers to create a PR in the future

I think the last 2 points don't really depend on the issue being closed, and usually, the bot will only close issues without a repro anyway.

Regarding the first point, while it shouldn't in an ideal scenario, that's not the case:

- having a lot of issues open makes it harder to focus and manage them, like having a lot of items in your todo list.
- people (and tools) count the number of open issues in a repo to determine the quality

Basically agree with @mikehardy
